src/core/trulens/core/utils/text.py

```diff
@@ -89,6 +89,31 @@ def format_quantity(quantity: float, precision: int = 2) -> str:
     return f"{result}{units[unit_idx]}"
 
 
+def format_size(size: int) -> str:
+    """Format a size (in bytes) into a human-readable string. This will use SI
+    prefixes. Implementation details are largely copied from
+    [millify](https://github.com/azaitsev/millify).
+
+    Args:
+        size: The quantity to format.
+
+    Returns:
+        str: The formatted quantity.
+    """
+
+    units = ["", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"]
+    unit_idx = max(
+        0,
+        min(
+            len(units) - 1,
+            int(math.floor(0 if size == 0 else math.log10(abs(size)) / 3)),
```

Comment:
haha are you expecting negative sizes or something?

Reply:
coped the code from the float one and didn't think enough about it